### PR TITLE
DPC-1585: Update implementer portal language

### PR DESCRIPTION
**Pull request by Sun-Mountain**
### Fixes [DPC-1585](https://jira.cms.gov/browse/DPC-1585)

Update implementer portal with final language by the H-Team

### Proposed Changes

- Changed language on log in page, landing page, profile editing page, and invited user account creation page.

### Change Details
- Removed demo information

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

### Acceptance Validation

Screen Shot 2021-07-08 at 11 12 02 AM

Screen Shot 2021-07-08 at 11 13 37 AM

Screen Shot 2021-07-08 at 11 13 51 AM

Screen Shot 2021-07-08 at 11 14 12 AM

Screen Shot 2021-07-19 at 1 03 54 PM

### CODE REVIEW

File: dpc-impl/app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -7,6 +7,10 @@
 
       <h1 class="ds-u-margin-top--0">Log in</h1>
 
+      <div>
+        <strong>This sandbox experience is designed for developers.</strong> If you are a healthcare provider interested in learning more about DPC, please see our Information for Providers for more information.
```
Comment (dhgreene) [inline dpc-impl/app/views/devise/sessions/new.html.erb:11]: Link?

File: dpc-impl/app/views/portal/index.html.erb [hunk 2]
```diff
@@ -1,16 +1,11 @@
 <% title "Team Members" %>
 
 <div id="dashboard__title">
-  <h1 class="ds-text-heading--4xl ds-title">My Organization Profile</h1>
-  <h1 class="ds-text-heading--xl ds-h3 ds-u-margin-top--1">for Implementer <%= @user.implementer %></h1>
-</div>
-
-<div class="ds-u-margin-y--3">
-  Donut dragée sweet roll. Lemon drops pie cake macaroon. Tiramisu toffee liquorice sweet carrot cake chocolate cake cake lemon drops. Bear claw gummies ice cream. Jelly beans sweet pudding biscuit cake brownie biscuit apple pie. Sesame snaps wafer muffin oat cake chupa chups cotton candy jujubes soufflé. Carrot cake jelly lemon drops. Icing bonbon pastry cake croissant halvah fruitcake. Oat cake tootsie roll sugar plum tart dragée topping. Macaroon wafer cookie cookie sweet roll topping chocolate chocolate bar.
+  <h1 class="ds-text-heading--xl ds-h3 ds-u-margin-top--1">Implementer <%= @user.implementer %></h1>
```
Comment (dhgreene) [inline dpc-impl/app/views/portal/index.html.erb:4]: Again, I'm a little uncomfortable using the word "Implementer" here

File: dpc-impl/app/views/devise/sessions/new.html.erb (hunk 1 above)
Comment (Sun-Mountain) [reply dpc-impl/app/views/devise/sessions/new.html.erb:11]: Link doesn't exist yet

File: dpc-impl/app/views/portal/index.html.erb (hunk 2 above)
Comment (Sun-Mountain) [reply dpc-impl/app/views/portal/index.html.erb:4]: I'm meeting with Crystal on Monday, will bring up these notes!

**Review by dhgreene — APPROVED** [review]
👍

**Pull request merged**